src/dams_mcda/server.R

```diff
@@ -366,48 +358,40 @@ server <- function(input, output, session) {
 			input$NumProperties4,
 			input$ElectricityGeneration4,
 			input$AvoidEmissions4,
-			input$IndigenousHeritage4,
+			input$IndigenousLifeways4,
 			input$IndustrialHistory4,
 			input$CommunityIdentity4,
-			input$Aesthetics4
+			input$Aesthetics4,
+			input$Health4,
+			input$Justice4
 			)
 
-		# create table matrix 1x5
-		Alt4_Table <- as.matrix(data.frame(Alt4))
-		row.names(Alt4_Table) <- criteria_names
-		names(Alt4_Table) <- "Raw Score"
-
-		# results
-		output$SummPlot4 <- renderBarPlot(

```

Comment:
We will want the graph generations here, but I have simplified them in my branch so deleting them here is fine

Reply:
ok I think that this might be resolved in the WSMUpdate_Dams --> TabsUpdate_Dams merge.